docs/advanced_topics/user_configuration.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: default
-nav_order: 7
-parent: Plug-in fundamentals
-title: User configuration & storage
+nav_order: 30
+parent: Advanced topics
+title: "User Configuration - Storage"
```

Comment:
User configuration - storage

And please remove the quotation marks. Or does this have a technical background? (I haven't looked at the rendered version yet).

Reply:
You need the quotation if you are using characters that have some other meaning like a ':'. Otherwise Jekyll will throw errors. I think I replaced the original ':' with the current dash, so the quotes are no longer necessary.